inst/@sym/coeffs.m

```diff
@@ -78,75 +79,96 @@
 %% @end group
 %% @end example
 %%
-%% Omitting the second output gives only the coefficients:
+%% Omitting the second output is not recommended, especially for non-interactive
+%% code, because it gives only the non-zero coefficients, and additionally
+%% the output is in the ``wrong order'' compared to other polynomial-related
+%% commands:
 %% @example
 %% @group
 %% c = coeffs (x^6 + 3*x - 4)
-%%   @result{} c = (sym) [1  3  -4]  (1×3 matrix)
+%%   @result{} c = (sym) [-4  3  1]  (1×3 matrix)
+%% @end group
+%% @end example
+%% @strong{Warning:} Again, note the order is reversed from the two-output
+%% case; this is for compatibility with Matlab's Symbolic Math Toolbox.
+%%
+%% If the optional input keyword @qcode{'all'} is passed, the zero
+%% coefficients are returned as well, and in the familiar order.
+%% @example
+%% @group
+%% c = coeffs (x^6 + 3*x - 4, 'all')
+%%   @result{} c = (sym) [1  0  0  0  0  3  -4]  (1×7 matrix)
 %% @end group
 %% @end example
-%% WARNING: Matlab's Symbolic Math Toolbox returns c = [-4 3 1]
-%% here (as of version 2014a).  I suspect they have a bug as its
-%% inconsistent with the rest of Matlab's polynomial routines.  We
-%% do not copy this bug.
+%% @strong{Note:} The @qcode{'all'} feature does not yet work with
+%% multivariate polynomials (https://github.com/cbm755/octsympy/issues/720).
 %%
 %% @seealso{@@sym/sym2poly}
 %% @end deftypemethod
 
-function [c, t] = coeffs(p, x)
+function [c, t] = coeffs(p, x, all)
 
-  if (nargin > 2)
```

Comment:
mm, if declare ```all=false;``` here and change it only when the conditions need it? (less lines)